components/global/gridHelper/GridHelper.tsx

```diff
@@ -0,0 +1,39 @@
+'use client'
+import { useEffect, useState } from 'react'
+
+// This component is used to show a grid overlay on the screen when the user presses cmd + g
+export default function GridHelper() {
+  const [isShowing, setIsShowing] = useState(false)
+
+  useEffect(() => {
+    window.addEventListener('keydown', handleKeydown)
+    return () => {
+      window.removeEventListener('keydown', handleKeydown)
+    }
+  }, [])
+
+  const handleKeydown = (event: KeyboardEvent) => {
+    if (event.metaKey && event.key === 'g') {
```

Comment:
This could possibly be conflicting with cases where we need to use the letter g and then it would be annoying if the grid accidentally shows up, right?
Maybe we could have an optional environment variable called `NEXT_PUBLIC_GRID_DISABLED` and if that is set (to true) then it is not possible to trigger the grid?